recipe/meta.yaml

```diff
@@ -110,6 +110,11 @@ outputs:
       run:
         - {{ pin_subpackage('kernel-headers_' ~ cross_target_platform, exact=True) }}
         - tzdata
+      run_constrained:
```

Comment:
This will probably need a repodata as otherwise, we will fallback to other build numbers, right?

Also, wouldn't it make sense to have a single `sysroot_linux_mutex` package to also cover the case where we add new architectures (like RISC-V)? All `sysroot_linux-*` packages would depend on it, and thus it would ensure consistency without the need for explicitly listing architectures here?